static/flow/views/program-editor-panel.js

```diff
@@ -336,18 +336,19 @@ var ProgramEditorPanel = function(options) {
                     var divindex = 1;
                     if(param.name=="period"){
                         displayedParamName = "last";
-                        $('<div>', {class: 'flowBlockParamLabel noSelect', html: displayedParamName}).appendTo(divflowBlockInputHolder);                    
-                        input = $('<input>', {class: 'form-control flowBlockInput', type: 'text', id: 'b' + block.id + '_bp_' + param.name, value: initval}).prependTo(divflowBlockInputHolder);
+                        $('<div>', {class: 'flow-block-param-label noSelect', html: displayedParamName}).appendTo(divflowBlockInputHolder);                    
+                        input = $('<input>', {class: 'form-control flow-block-input', type: 'text', id: 'b' + block.id + '_bp_' + param.name, value: initval}).appendTo(divflowBlockInputHolder);
```

Comment:
Just curious, why change the `prepend`s to `append`s?

Reply:
Not sure why they were prepends in the first case (copy-paste fossil maybe?), but the issue I had was that the prepend made the element order inside of the parent container confusing during debugging.  So visually we had a container with elements ordered A,B,C, but when we used the browser dev tools to view the elements in the container, things were organized C,A,B.  Very small, but it threw me off for a minute.  I'm hoping this saves someone a little time in the future because the visual order and the internal element order will now be in sync.

Reply:
Definitely makes sense, good call!